From 93736372a66cc6d35c2b848fd710b39b40293f0d Mon Sep 17 00:00:00 2001
From: Roman Vais <rvais@redhat.com>
Date: Wed, 14 Jun 2017 14:00:28 +0200
Subject: [PATCH] Remove Brotli compressor

---
 .../commons/compress/compressors/CompressorStreamFactory.java    | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/src/main/java/org/apache/commons/compress/compressors/CompressorStreamFactory.java b/src/main/java/org/apache/commons/compress/compressors/CompressorStreamFactory.java
index 4bd22aa..238257d 100644
--- a/src/main/java/org/apache/commons/compress/compressors/CompressorStreamFactory.java
+++ b/src/main/java/org/apache/commons/compress/compressors/CompressorStreamFactory.java
@@ -31,8 +31,6 @@ import java.util.Set;
 import java.util.SortedMap;
 import java.util.TreeMap;
 
-import org.apache.commons.compress.compressors.brotli.BrotliCompressorInputStream;
-import org.apache.commons.compress.compressors.brotli.BrotliUtils;
 import org.apache.commons.compress.compressors.bzip2.BZip2CompressorInputStream;
 import org.apache.commons.compress.compressors.bzip2.BZip2CompressorOutputStream;
 import org.apache.commons.compress.compressors.deflate.DeflateCompressorInputStream;
@@ -540,13 +538,6 @@ public class CompressorStreamFactory implements CompressorStreamProvider {
             if (BZIP2.equalsIgnoreCase(name)) {
                 return new BZip2CompressorInputStream(in, actualDecompressConcatenated);
             }
-            
-            if (BROTLI.equalsIgnoreCase(name)) {
-                if (!BrotliUtils.isBrotliCompressionAvailable()) {
-                    throw new CompressorException("Brotli compression is not available.");
-                }
-                return new BrotliCompressorInputStream(in);
-            }
 
             if (XZ.equalsIgnoreCase(name)) {
                 if (!XZUtils.isXZCompressionAvailable()) {
-- 
2.7.4

